### PR TITLE
To do list Pull Request

### PR DESCRIPTION
**Pull request by henask12**
# Pull Request: Task Management and Task Completion UI

>  This pull request creates task management and task completion UI to the to-do list application. The key features introduced in this update are:

- [ ] `Task Management`: The application allows users to manage tasks by displaying a list of tasks.

- [ ]  `Task Completion`: Each task in the list is accompanied by a checkbox that indicates whether the task has been completed or not. Users can toggle the checkbox to mark a task as completed or incomplete.

These features enhance the usability and effectiveness of the to-do list application, providing users with better control over their tasks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,130 +1,77 @@
...
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
-- [Live Demo Link](https://google.com)
+> There is no live demo for now stay tunned!
```
Comment (Ismailco) [inline README.md:66]: - [ ] Kindly make sure to remove the `Live demo` section altogether if you're not planning on deploying the project yet

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @henask12, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ Used Webpack
✔️ Clean code
✔️ 
✔️ No linter errors
✔️ Used GitHub Flow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -240,52 +141,39 @@ Example:
...
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LICENSE -->
 
 ## 📝 License <a name="license"></a>
 
-This project is [MIT](./LICENSE) licensed.
-
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+This project is [MIT](./MIT) licensed.
```
Comment (TedLivist) [inline README.md:177]: - [ ] You are yet to fix the MIT link. When I click it I should be taken to the MIT.md file but instead I am led to this. Kindly fix that.

  image

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @henask12,

✔️ App is set up with webpack

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (henask12) [reply README.md:177]: @TedLivist you are checking the master branch 
on the todolist branch it is already fixed

File: .eslintrc.json
Comment (AbelG101) [inline .eslintrc.json:1]: - [ ] [OPTIONAL] Hey Henok, For your future reference I've left you some comments regarding the contents in your license file on the screenshot below 👇 so you can check them out.
    image

**Review by AbelG101 — APPROVED** [review]
# Hi @henask12 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- UI is looking great ✔
- Well-structured files ✔
- All linter checks are passing ✔
- Good PR title and summary ✔

## Optional suggestions
- Please check the comments below...

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .eslintrc.json
Comment (henask12) [reply .eslintrc.json:1]: @AbelG101 Thank you for the reference

**Pull request merged**